scripts/register_saft_addresses.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Register SAFT accounts script
+ *
+ * ts-node scripts/register_saft_addresses.ts "network" "path to csv with addresses"
+ */
+
+import fs from 'fs'
+import { utils, constants, providers, Wallet } from 'ethers'
+import { TimeLockRegistry } from '../build/types/TimeLockRegistry'
+import { TimeLockRegistryFactory } from '../build/types/TimeLockRegistryFactory'
+import { TrustToken } from '../build/types/TrustToken'
+import { TrustTokenFactory } from '../build/types/TrustTokenFactory'
+
+export const txnArgs = { gasLimit: 2_000_000, gasPrice: 20_000_000_000 }
+
+export interface SaftAccount {
+  address: string,
+  amount: string,
+}
+
+const toTrustToken = (amount: string) => utils.parseUnits(amount, 8)
```

Comment:
`toTrustToken`, but from what? Maybe sth like `weiToTrustToken`?

Reply:
The problem is, can we call it wei when we don't have 18 digits precision? It's kinda weird that community hasn't come with any universal name for minimal token unit

Reply:
Right... Quantum sounds like a proper word, but using it might actually cause even more confusion as it's not a common idea. So maybe let's keep `toTrustToken`, but move it to some shared utils directory and import from there. And actually have the version with `BigNumberish` instead of `string`.